configure

```diff
@@ -165,8 +167,9 @@ def main():
         if config.target.sdk == OSType.MacOSX and config.system_root is None and Target(Target.default()).sdk == OSType.MacOSX:
             import subprocess
             config.system_root = Path.path(subprocess.Popen(['xcrun', '--show-sdk-path'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE).communicate()[0])
-            swift_path = Path.path(subprocess.Popen(['xcrun', '--find', 'swift'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE).communicate()[0]).parent().parent()
-            config.swift_sdk = swift_path.absolute()
+            if config.swift_sdk is None:
+                swift_path = Path.path(subprocess.Popen(['xcrun', '--find', 'swift'], stdout=subprocess.PIPE, stderr=subprocess.PIPE, stdin=subprocess.PIPE).communicate()[0]).parent().parent()
+                config.swift_sdk = swift_path.absolute()
```

Comment:
I did split it in to 2 commits - the first one just fixes things up for OSX but doesn't rename, so you can build the dylib as part of the usual swift build script, but it won't install due to name conflict with the SDK overlay.